acceptance_tests/features/rh_ui_cookie_selection.feature

```diff
@@ -0,0 +1,50 @@
+@UI
+Feature: testing the cookie selection functionality of RH UI
+
+    Scenario: The cookies banner is displayed
+        When the UAC entry page is displayed
+        Then the cookies banner is displayed
+
+    Scenario: The 'cookies' hyperlink on the cookies banner points to the cookies page
+        Given the UAC entry page is displayed
+        And the cookies banner is displayed
+        Then the 'cookies' hyperlink on the cookies banner points to en/cookies/
+
+    Scenario: The "View cookies" hyperlink on the cookies banner points to the cookies page
+        Given the UAC entry page is displayed
+        And the cookies banner is displayed
+        Then the 'View cookies' hyperlink points to en/cookies/
+
+    Scenario: The "change cookie preferneces" hyperlink points to the cookies page
+        Given the UAC entry page is displayed
+        And the cookies banner is displayed
+        When the user accepts the cookies on the cookies banner
+        Then the "change cookie preferences" hyperlink text points to en/cookies/
```

Comment:
I feel like these don't need to be separated when nothing is changing with the `given` and `when`. You just have one scenario checking that all the links go to the correct places